docs/core/tutorials/debugging-with-visual-studio.md

```diff
@@ -28,54 +28,68 @@ You should always begin by testing your program in Debug mode. Debug mode turns
 
-# [C#](#tab/csharp)
 1. Enter `date = new DateTime(2016,11,01,11,59,00)` in the **Immediate Window** and press the Enter key.
 
    The **Immediate Window** displays the value of the string variable and the properties of the <xref:System.DateTime> value. In addition, the value of the variables is updated in the **Autos** and **Locals** windows.
 
    ![Autos window and Immediate Window](./media/debugging-with-visual-studio/autosimmediate.png)
-# [Visual Basic](#tab/visual-basic)
+
+1. Continue program execution by selecting the **Continue** button in the toolbar or by selecting the **Debug** > **Continue** menu item. The values displayed in the console window correspond to the changes you made in the **Immediate Window**.
+
+   ![Console window showing the typed value Jack at the What is your name? prompt followed by Hello Gracie on 11/1/2016 at 11:59am](./media/debugging-with-visual-studio/changed.png)
+
+1. Press any key to exit the application and end Debug mode.
+# [Visual Basic](#tab/vb)
+1. A *breakpoint* temporarily interrupts the execution of the application *before* the line with the breakpoint is executed. 
+
+   Set a breakpoint on the line that reads `Console.WriteLine(vbCrLf + $"Hello, {name}, on {currentDate:d} at {currentDate:t}!")` by clicking in the left margin of the code window on that line or by choosing the **Debug** > **Toggle Breakpoint** menu item with the line selected. As the following figure shows, Visual Studio indicates the line on which the breakpoint is set by highlighting it and displaying a red circle in its left margin.
+
+   ![Visual Studio Program window with breakpoint set](./media/debugging-with-visual-studio/vb-setbreakpoint.png)
+
+1. Run the program in Debug mode by selecting the **HelloWorld** button with the green arrow on the toolbar, pressing F5, or choosing **Debug** > **Start Debugging**.
+
+1. Enter a string in the console window when the program prompts for a name and press Enter.
+
+1. Program execution stops when it reaches the breakpoint and before the `Console.WriteLine` method executes. The **Autos** window displays the values of variables that are used around the current line. The **Locals** window (which you can view by clicking the **Locals** tab) displays the values of variables that are defined in the currently executing method.
+
+   ![Visual Studio application window](./media/debugging-with-visual-studio/vb-break.png)
+
+1. You can change the value of the variables to see how it affects your program. If the **Immediate Window** is not visible, display it by choosing the **Debug** > **Windows** > **Immediate** menu item. The **Immediate Window** lets you interact with the application you're debugging.
+
+1. You can interactively change the values of variables. Enter `name = "Gracie"` in the **Immediate Window** and press the Enter key.
+
 1. Enter `currentDate = new DateTime(2016,11,01,11,59,00)` in the **Immediate Window** and press the Enter key.
```

Comment:
C# and VB are not matching at this point and VB starts numbering again:
![image](https://user-images.githubusercontent.com/12971179/34132967-3d375c64-e407-11e7-8e6d-ce6e1908c121.png)

Seems wrong.